data-processing-lib/python/src/data_processing/transform/table_transform.py

```diff
@@ -99,20 +98,18 @@ def flush(self) -> tuple[list[pa.Table], dict[str, Any]]:
         """
         return [], {}
 
-    @staticmethod
-    def _check_and_convert_tables(
-            out_tables: list[pa.Table], stats: dict[str, Any]
-    ) -> tuple[list[tuple[bytes, str]], dict[str, Any]]:
+    def _check_and_convert_tables(self, out_tables: list[pa.Table], stats: dict[str, Any]) \
```

Comment:
are you missing pre-commit?

Reply:
Jus did.
It created a massive amount of changes